carousel/core/calculators.py

```diff
@@ -186,13 +186,14 @@ def calculate(cls, calc, formula_reg, data_reg, out_reg,
         fargs = formula_reg.args.get(formula, [])  # formula arguments
         constants = formula_reg.isconstant.get(formula)  # constant args
         # formula arguments that are not constant
-        vargs = [a for a in fargs if a not in constants]
+        vargs = [] if constants is None else [a for a in fargs if a not in constants]
```

Comment:
before this was protected inside `if constants is not None`, but since I removed this, I need the extra check here :frowning_face: